R/test.data.table.R

```diff
@@ -69,6 +71,24 @@ compactprint <- function(DT, topn=2L) {
 
 INT = function(...) { as.integer(c(...)) }   # utility used in tests.Rraw
 
+ps_mem = function() {
+  if (!identical(.Platform$OS.type, "unix")) {
+    warning("data.table internal function 'ps_mem' is designed to work only on Linux, please upgrade your OS and re-run.")
```

Comment:
'please upgrade your OS [to Linux]' ... LOL.  I know I've written similar in the past, but I vaguely remember a complaint so I changed it. Better just to remove that part. Also, it's not technically correct because this will work on MacOS too, won't it?  And it should work on Windows 10 where native Linux tools are now included.  To be fully correct, it may be worth checking if `ps` system command with the parameters needed is available and functioning as expecting and if not, then error message would be "this OS does not support `ps -o rss <pid>`. It may be possible to install ps".